test/ctia/task/migration/store_test.clj

```diff
@@ -123,6 +294,154 @@
 (def fixtures-nb 100)
 (def examples (fixt/bundle fixtures-nb false))
 
+(deftest sliced-queries-test
+  (let [storemap {:conn es-conn
+                  :indexname "ctia_relationship"
+                  :mapping "relationship"
+                  :props {:write-index "ctia_relationship"}
+                  :type "relationship"
+                  :settings {}
+                  :config {}}
+        data (es-helpers/load-file-bulk es-conn "./test/data/indices/sample-relationships-1000.json")
```

Comment:
What is the difference between adding 1000 relationships or 19 relationships to test this function?

Reply:
I have selected these relationships among 4 millions to be sufficiently random and representative (including entities with missing `modified`field). 
I checked with`time`, it takes 300ms to load these relationships with bulk, and the whole test takes 400ms.

Reply:
What is the purpose of this test? How I know that that each case is tested? If the format the Relationship entity change how can I generate a new file? IMO we should not use data like this for a unit test.

Reply:
the only thing it depends on is the modified field. whatever will be future format of relationship, if it keeps `modified` metadata, which is controlled by CTIA, it is it's OK.